include/solvers/navier_stokes_vof_assemblers.h

```diff
@@ -40,8 +41,10 @@ class GLSNavierStokesVOFAssemblerCore : public NavierStokesAssemblerBase<dim>
 {
 public:
   GLSNavierStokesVOFAssemblerCore(
-    std::shared_ptr<SimulationControl> simulation_control)
+    std::shared_ptr<SimulationControl> simulation_control,
+    const SimulationParameters<dim> &  nsparam)
     : simulation_control(simulation_control)
+    , vof_parameters(nsparam.multiphysics.vof_parameters)
```

Comment:
It's a small detail, maybe this could be added in the next PR as you mentionned, but there should only be one parameter being the ns_param, and then simulation_control should be initialized like : simulation_control(ns_param.simulation_control) or something like this.

Reply:
good catch! I'll do this on this PR :)

Reply:
Okay awesome! :)